stylesheets/global-style.css

```diff
@@ -0,0 +1,374 @@
+@charset "UTF-8";
+
+body {
+  text-align: center;
+  color: black;
+	margin-top: 0px;
+	margin-right: 0px;
+	margin-bottom: 0px;
+	margin-left: 0px;
+	font-style: normal;
+	font-weight: 200;
+}
+
+img {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  padding: 0;
+  display: block;
+  margin: 0 auto;
+  max-width: 100%;
+}
+
+html, body {
+    height: 100%;
+    margin: 0;
+    padding: 0;
+}
+
+/* Header */
+header {
+  background-color: #2952a3;
+  color: #99CC33;
+}
+
+.total {
+  color: white;
+}
+
+.header-container {
+  width: 100%;
+  height: 100px;
+  padding-bottom: 50px;
+  background-color: #193366;
+  float: left;
+  clear: none;
+  margin: 0;
+}
+
+.header-container-dogs {
+  width: 100%;
+  height: 100px;
+  padding-bottom: 50px;
+  background-color: #193366;
+  margin: 0;
+}
+
+.header-container-blog {
+  width: 100%;
+  height: 100px;
+  padding-bottom: 50px;
+  background-color: #193366;
+  margin: 0;
+}
+
+.header-container-checkout {
+  width: 100%;
+  height: 100px;
+  padding-bottom: 50px;
+  background-color: #193366;
+  margin: 0;
+}
+
```

Comment:
Somewhat unrelated, but I would adjust the spacing for the nav, links and content to have more space on the left and right of the page for desktop layouts similar to the mockups. Currently your content extends to the edges of the screen. Roughly 10% - 15% on each side I would guess from the pdf.